oauth2.gemspec

```diff
@@ -5,7 +5,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'oauth2/version'
 
 Gem::Specification.new do |spec|
-  spec.add_dependency 'faraday', ['>= 0.8', '< 0.15.0']
+  spec.add_dependency 'faraday', ['>= 0.8', '<= 0.15.1']
```

Comment:
Please don't do this. Use at least `< 0.16.0` (as it was before) so when a new bugfix-version 0.15.2 is released you don't need to change and release a new version again.

Reply:
This is a POC PR from depfu, which does a kind of Gemfile spot check upgrading, to see if the build will pass with a very specific version constraint.  We won't be merging it.  No worries!

Reply:
Yeah, a hard less than requirement on the current minor version is really frustrating for upstream gems that depend on oauth2.

Reply:
@Ch4s3 @SuperTux88 This PR, or one like it, will be used as a basis to update, and possibly loosen, the faraday dependency.  We are working on a new release, but all maintainers are time constrained.  If someone using this gem has time to switch their local dev env to their own fork, and make the changes, and validate the functionality, and push a PR that updates all the appraisal gemfiles and the gemspec, and update specs... that would be awesome.

Reply:
@pboling @Ch4s3 @SuperTux88 take a look @ #385.  It works for us.